java/Glacier2/greeter/settings.gradle.kts

```diff
@@ -19,3 +19,5 @@ dependencyResolutionManagement {
 rootProject.name = "greeter"
 include("client")
 include("server")
+
+includeBuild("../../config/build-logic")
```

Comment:
Tells Gradle that it needs to build our local convention-plugin, before it can build the rest of this project.

Reply:
We can do anything of course, but `config` is the expected name for checkstyle and rewrite (when it's not in the project root at least). So, if we want to still follow conventions, we could do two directories instead:
```
build-common/
    all-the-plugin-stuff/
        ...
    config/
        rewrite.yml
        checkstyle
            checkstyle.xml
            suppressions.xml
```
Fine with you?

Reply:
Or maybe just this, to eliminate an extra layer of directories to go through. And so that checkstyle and rewrite are still in their expected places (since the "project root" of the demos is logically the `java` folder):
```
build-common/
    ... all-the-plugin-stuff/
config/
    rewrite.yml
    checkstyle/
        checkstyle.xml
        suppressions.xml
```